common/pom.xml

```diff
@@ -179,7 +179,8 @@ under the License.
             </goals>
             <configuration>
               <sources>
-                <source>src/main/${shims.source}</source>
+                <source>src/main/${shims.majorSource}</source>
+                <source>src/main/${shims.minorSource}</source>
```

Comment:
Thanks for adding the minor version shims.  These are going to help me with some of work around supporting `cast`.